recipe/meta.yaml

```diff
@@ -9,12 +9,13 @@ source:
   sha256: c739578bf6b764aa0752db9a2fdadcfe921c78f1228c7ec0bb47fa804c55d17b
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage('jansson', max_pin='x') }}
 
 requirements:
   build:
+    - gnuconfig   # [unix]
     - automake  # [not win]
     - autoconf  # [not win]
```

Comment:
Maybe worth converting these to `# [unix]` as well for consistency

Reply:
Updated! Thanks for the suggestion.